storage/grpc_writer.go

```diff
@@ -102,7 +102,7 @@ func (c *grpcStorageClient) OpenWriter(params *openWriterParams, opts ...storage
 	setFlush(func() (int64, error) {
 		return gw.flush()
 	})
-	gw, err := newGRPCWriter(c, s, params, pr, pw, params.setPipeWriter)
+	gw, err := newGRPCWriter(c, s, params, pr, pr, pw, params.setPipeWriter)
```

Comment:
Feels redundant to send this in twice

Reply:
I agree but unfortunately gax.DetermineContentType returns a reader and therefore it's not possible to just switch to Io.PipeReader throughout (I tried it...)